### ISSUE TITLE
leverage RFC 9250

### ISSUE THREAD

**Issue by boucadair** [netconf-wg/netconf-over-quic#6]
Although I expect the mapping to be less complex, I encourage the authors to look into RFC 9250 and leverage it as much as appropriate.

**Comment by marcblanchet** [comment]
I agree that RFC9250 is well written and comprehensive. However, DNS has more complicated stuff than NETCONF regarding its transport. Not sure what to do here, unless more specific advice is given.
